tools/glide-pindeps/glide/util.go

```diff
@@ -0,0 +1,41 @@
+package glide
+
+import (
+	"fmt"
+	"os"
+)
+
+// MissingImports receives a LockFile and a YamlFile and returns a slice of
+// imports not contained in the YamlFile. Dependencies with a non-empty "repo" field
+// are skipped, as they are assumed to contain forked dependencies.
+func MissingImports(lockfile *LockFile, yamlfile *YamlFile) ([]*YamlFileImport, error) {
+	if lockfile == nil || yamlfile == nil {
+		return nil, fmt.Errorf("both a lockfile and a yamlfile are required")
+	}
+
+	newImports := []*YamlFileImport{}
+
+	for _, lockDep := range lockfile.Imports {
+		if len(lockDep.Repo) > 0 {
+			fmt.Fprintf(os.Stderr, "info: skipping package with \"repo\" field: %s\n", lockDep.Name)
```

Comment:
Not from a library method.  I don't like libraries to log.  Instead, return all of these as a third value.